--- origsrc/sylpheed-2.7.1/libsylph/Makefile.am	2009-08-12 20:59:08.000000000 -0500
+++ src/sylpheed-2.7.1/libsylph/Makefile.am	2009-08-25 18:32:04.961983600 -0500
@@ -97,13 +97,12 @@ EXTRA_DIST = \
 	libsylph-0.def
 
 if NATIVE_WIN32
-no_undefined = -no-undefined
 export_symbols = -export-symbols libsylph-0.def
 endif
 
 libsylph_0_la_LDFLAGS = \
 	-version-info $(LT_CURRENT):$(LT_REVISION):$(LT_AGE) \
-	-export-dynamic $(no_undefined) $(export_symbols)
+	-export-dynamic -no-undefined $(export_symbols)
 
 libsylph_0_la_LIBADD = $(GLIB_LIBS)
 
--- origsrc/sylpheed-2.7.1/src/Makefile.am	2009-08-12 20:59:30.000000000 -0500
+++ src/sylpheed-2.7.1/src/Makefile.am	2009-08-25 18:33:36.717135500 -0500
@@ -112,13 +112,12 @@ libsylpheed_plugin_0include_HEADERS = \
 	plugin.h
 
 if NATIVE_WIN32
-no_undefined = -no-undefined
 export_symbols = -export-symbols libsylpheed-plugin-0.def
 endif
 
 libsylpheed_plugin_0_la_LDFLAGS = \
 	-version-info $(LT_CURRENT):$(LT_REVISION):$(LT_AGE) \
-	-export-dynamic $(no_undefined) $(export_symbols)
+	-export-dynamic -no-undefined $(export_symbols)
 
 libsylpheed_plugin_0_la_LIBADD = \
 	$(GTK_LIBS) \
